src/main/java/de/bonndan/nivio/config/NivioConfigProperties.java

```diff
@@ -36,6 +36,9 @@
 
     @Pattern(regexp = URL_REGEX, message = "brandingLogoUrl must be a valid URL")
     private String brandingLogoUrl;
+
+    @Pattern(regexp = "[a-zA-Z ]+", message = "brandingMessage must be a valid string")
```

Comment:
This is a bit restrictive, but since there were no clear requirements, it is ok 👍

Reply:
okay